src/makeDefaultEffects.js

```diff
@@ -41,13 +41,20 @@ export default function makeDefaultEffects() {
   effects[HTTP] = function(props, getAction) {
     props.options = props.options || {}
     props.options.response = props.options.response || "json"
+    var errorAction = props.options.error || props.action
```

Comment:
Not sure it's intuitive that the same action would be handling errors as success by default. I'd rather have error handling be explicit.

If we took this approach does it mean that actions called by the `http` effect should look at their payload to see if it's an error or not?

Reply:
Take a look at the discussion with @Mytrill (he had the same question). Otherwise, why would be behavior expected when the user doesn't provide any handler for error ?

Reply:
@titouancreach I'm not sure if it's worse to drop errors completely or potentially cause runtime errors from actions that weren't designed to handle them 🤔